camel/source/pkg/apis/sources/v1alpha1/camelsource_types.go

```diff
@@ -22,6 +22,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"knative.dev/pkg/apis/duck"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
 	duckv1alpha1 "knative.dev/pkg/apis/duck/v1alpha1"
```

Comment:
perhaps using `knative.dev/pkg/apis` instead of the alpha1 ?

Reply:
Done, thanks